### ISSUE TITLE
about tfrecords

### ISSUE THREAD

**Issue by luchen828** [timctho/convolutional-pose-machines-tensorflow#75]
hi,@ timctho 
1.do you konw the numbers of data in the label txt? 
2.is the name of jpg + 4 coordinates of hand bbox + 21 coordinates of hand keypoint? 
3.the pretrained model can be cpm_hand.pkl?
when i read the tfrecords there has some problem,hope you can reply me .thank you very much
